Source/Core/Core/PowerPC/PowerPC.cpp

```diff
@@ -546,14 +550,49 @@ void CheckExternalExceptions()
   }
 }
 
-void CheckBreakPoints()
+static u32 TriggerBreakPoint(const TBreakPoint& breakpoint)
```

Comment:
This can still return as a bool, since it's not used in JITed code, with the cast being performed where necessary.